src/string/strlen.rs

```diff
@@ -3,12 +3,44 @@ use core::usize;
 use c_types::*;
 
 #[no_mangle]
-pub unsafe extern "C" fn strlen(s: *const c_schar) -> size_t {
-    // TODO(adam) convert to checking word-size chunks
-    for i in 0.. {
-        if *s.offset(i) == 0 {
-            return i as usize;
+pub unsafe extern "C" fn strlen(mut s: *const c_schar) -> size_t {
+    
+    #[inline]
+    fn align() -> usize { core::mem::size_of::<size_t>() }
```

Comment:
Done that.
But I'm not sure how it fix the problem.

I see `core::usize` is at the top of this `strlen.rs` originally, so I guess `::core` is already available?

I should check the project structure at first though, I'll spend some time to investigate.